labelbox/schema/project.py

```diff
@@ -1360,6 +1360,7 @@ def __check_data_rows_have_been_processed(
 class ProjectMember(DbObject):
     user = Relationship.ToOne("User", cache=True)
     role = Relationship.ToOne("Role", cache=True)
+    access_from = Field.String("accessFrom")
```

Comment:
I assume this attribute should be snake case instead of `accessFrom`

Reply:
oops, yes. Sorry for the wrong info, I haven't contributed to the sdk in a while.

Reply:
I think the string arg passed to the Field.String constructor should also be came case.

Reply:
updated